### PR TITLE
Support only list

### LINKED ISSUE #19: Implement other collections in Node.transformChildren or throw exception

### PR DESCRIPTION
**Pull request by matozoid**
Fixes #19. Maybe not a great solution, but better than a TODO :-)

### CODE REVIEW

File: src/main/kotlin/com/strumenta/kolasu/model/Processing.kt [hunk 1]
```diff
@@ -229,7 +229,7 @@ fun Node.transformChildrenInPlace(operation: (Node) -> Node) {
                         }
                     }
                 } else {
-                    TODO()
+                    throw UnsupportedOperationException("Only modifications in a List and MutableList are supported, not " + value::class.toString())
```
Comment (ftomassetti) [inline src/main/kotlin/com/strumenta/kolasu/model/Processing.kt]:
```suggestion
                    throw UnsupportedOperationException("Only modifications in a List and MutableList are supported, not " + value::class.toString())
```

```suggestion
                    throw UnsupportedOperationException("Only modifications in a List and MutableList are supported, not ${value::class.toString()}")
```

Kotlin has string interpolation :)

Comment (matozoid) [reply src/main/kotlin/com/strumenta/kolasu/model/Processing.kt]: Stupid Intellij didn't suggest it, so I was lazy :-)

Comment (matozoid) [reply src/main/kotlin/com/strumenta/kolasu/model/Processing.kt]: Aaaand I forgot to do this :-D

Comment (ftomassetti) [reply src/main/kotlin/com/strumenta/kolasu/model/Processing.kt]: I changed it and merged it

**Pull request merged**